packages/rich-text/src/component/index.js

```diff
@@ -325,8 +325,32 @@ function RichText(
 			multilineTag,
 			preserveWhiteSpace,
 		} );
-		event.clipboardData.setData( 'text/plain', plainText );
-		event.clipboardData.setData( 'text/html', html );
+
+		const isLi = /<li>.*<\/li>/gi;
+		const foundLi = html.match( isLi );
+		if ( foundLi ) {
+			let liCopyString = html
+				.split( '<li>' )
+				.join( '' )
+				.split( '</li>' )
+				.filter( Boolean );
+			const liCopyStringLength = liCopyString.length - 1;
+			liCopyString = liCopyString.map( ( el, i ) => {
+				if ( el && i === 0 ) {
+					return '<ul><li>' + el + '</li>';
+				} else if ( el && i === liCopyStringLength ) {
+					return '<div style="clear:both"/><li>' + el + '</li></ul>';
```

Comment:
The direct children of <ul> shold be only <li> and not <div>. The generated HTML would be sematicly incorrect.

Reply:
according to yours link I can paste template tag , I need something between li tags to push them to new lines

Reply:
cant you just style li elements? Btw, it is better not to add inline styles, but CSS classes.

Reply:
> cant you just style li elements? Btw, it is better not to add inline styles, but CSS classes.

I ve tested this approach , it must be something between li elements